internal/proxy/segment.go

```
@@ -319,10 +319,7 @@ func (sa *segIDAssigner) syncSegments() (bool, error) {
 
 	log.Debug("syncSegments call dataCoord.AssignSegmentID", zap.Stringer("request", req))
 
-	resp, err := sa.dataCoord.AssignSegmentID(context.Background(), req)
-	if err != nil {
-		return false, fmt.Errorf("syncSegmentID Failed:%w", err)
-	}
```

We cannot ignore this error since it might be network issue here